### ISSUE TITLE
Add Manifest v3 support for Firefox

### ISSUE THREAD

**Issue by markmhendrickson** [blockstack/blockstack-app#2812]

**Comment by kyranjamie** [comment]
MV3 support anticipated early 2023

**Comment by paboum** [comment]
> MV3 support anticipated early 2023

is that really the best estimate?

**Comment by markmhendrickson** [comment]
Our support for Firefox is now deprecated

**Comment by paboum** [comment]
still building firefox releases though?

**Comment by markmhendrickson** [comment]
Yep they can be found here for installation from source: https://github.com/leather-io/extension/releases

We just don't support or make any assurances about them functioning correctly.
